js/index.js

```diff
@@ -24,7 +24,7 @@ $("#howmany").hide();
 // The sticky becomes draggable and can be moved onto the "brainstorm-board"
 $('#clone-button').click(function(){
 	$('<p></p>').appendTo($('#sticky-generator'))
-	var newSticky = $('.sticky-template').last().clone();
```

Comment:
@msweinberg I found breaking this up into multiple lines helped me debug it. Chaning this

```javascript
var newSticky = $('.sticky-template').last().clone();
``` to 
```javascript
var lastNode = $('.sticky-template').last();
var newSticky = lastNode.clone();
```
This made it easier for me to see in the debugger that `last` wasn't getting the original template node.